test/__snapshots__/mathml-spec.js.snap

```diff
@@ -42,7 +42,7 @@ exports[`A MathML builder accents turn into <mover accent="true"> in MathML 1`]
           e
         </mi>
         <mo>
-          ´
```

Comment:
This changed b/c we switched some of the accents to be in the "Spacing Modifier Characters" Unicode block.  See https://en.wikipedia.org/wiki/Spacing_Modifier_Letters.